index.js

```diff
@@ -167,6 +169,7 @@ var validator = {
   matches: _matches.default,
   isEmail: _isEmail.default,
   isURL: _isURL.default,
+  isAnyURI: _isAnyURI.default,
```

Comment:
Could you separate it out this new checker, `isAnyURI` as a different PR?

Reply:
yes, I didn't mean for that to be on this branch. anyURI isn't done yet.